bril-txt/briltxt.py

```diff
@@ -50,13 +51,21 @@
 
 class JSONTransformer(lark.Transformer):
     def start(self, items):
+        imports = []
+        while len(items) > 0 and type(items[0]) == lark.lexer.Token:
```

Comment:
can we change this to:
while len(items) ...

Reply:
I think `len(items) > 0` is cleaner, makes the intent of the condition more obvious.